1-exercises/B-setting-values/exercise1.js

```diff
@@ -21,6 +21,12 @@ let capitalCities = {
     - Add a name of "Lima" to Peru's capital city.
     - Add a population of 9750000 to Peru's capital city.
 */
+capitalCities.UnitedKingdom.name = 8980000;
```

Comment:
should the UK be renamed to 8980000? bit of a weird name for a country if you ask me ;)